core/rfb.js

```diff
@@ -1237,6 +1237,7 @@ export default class RFB extends EventTargetMixin {
         encs.push(encodings.pseudoEncodingXvp);
         encs.push(encodings.pseudoEncodingFence);
         encs.push(encodings.pseudoEncodingContinuousUpdates);
+        encs.push(encodings.pseudoEncodingVMwareCursor);
```

Comment:
I think we need to only advertise this encoding when `this._fb_depth` is `24`, just like for `pseudoEncodingCursor` as we assume a certain pixel format in the incoming data (for classic cursors at least).

Reply:
Good point. Fixed now